src/pages/public_registration/action_accordions/RefreshPublishingRequestButton.tsx

```diff
@@ -0,0 +1,24 @@
+import RefreshIcon from '@mui/icons-material/Refresh';
+import { LoadingButton } from '@mui/lab';
+import { t } from 'i18next';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+interface RefreshPublishingRequestButtonProps {
+  refetchData: () => void;
+  isLoadingData: boolean;
+}
+export const RefreshPublishingRequestButton = ({ refetchData, isLoadingData }: RefreshPublishingRequestButtonProps) => {
+  return (
+    <LoadingButton
+      variant="contained"
+      color="info"
+      size="small"
+      loading={isLoadingData}
```

Comment:
Når vi endrer denne knappen er det kanskje greit å fjerne oversettelsen at isLoadingData=loading og gå over til den nye måten å bruke loading på med vanlig `Button`?

```suggestion
interface RefreshPublishingRequestButtonProps extends ButtonProps {
  refetchData: () => void;
}

export const RefreshPublishingRequestButton = ({ refetchData, ...buttonProps }: RefreshPublishingRequestButtonProps) => {
  return (
    <Button
      {...buttonProps}
      variant="contained"
      color="info"
      size="small"
```